### DISCUSSION TITLE
[bug]: Recover funds after the channel was force closed from the remote side

### DISCUSSION THREAD

**Discussion by bcoufal** [lightningnetwork/lnd#9181] [Troubleshooting] [answered]
### Background

I wanted to open a channel from the Hydranode to the Kappa node. Unfortunately, I entered too low a fee and it looked like the transaction would be confirmed at infinity. When I found this out, I tried to delete the transaction from the mempool, but unfortunately, it only deleted the "pending channel" information from the LND on Hydranode. I am now in a state where the transaction https://mempool.space/cs/tx/47b157b734c4ed624c4c378331c484a92c924b6dbb3659dea657ade7256269a3 were confirmed (I have done acceleration of it in cooperation with mempool.space) and I did ask the operator of Kappa (Aaron LRoS - WoS <[support@walletofsatoshi.com](mailto:support@walletofsatoshi.com)>) to force close the channel from his side. After this step, the fund moved to https://mempool.space/cs/address/bc1qk7tvtq92nc0h2swtfhf9afkjuvk5djyj5dcqk2zxj5tr2nfm9p5st8qpjv, but I can't see it in our LND node wallet. I did rescan the wallet but that step had no effect.

I did try to cancel this channel by command "lncli abandonchannel ...".

### Your environment

* version of `lnd`
lnd version 0.18.1-beta commit=v0.18.1-beta
* which operating system (`uname -a` on *Nix)
Linux HYDRAnode 5.10.0-32-amd64 #1 SMP Debian 5.10.223-1 (2024-08-10) x86_64 GNU/Linux
* version of `btcd`, `bitcoind`, or other backend
Bitcoin Core version v27.1.0

**Comment by ziggie1984** [comment, marked as answer]
You should never call the cmd `abandonchannel` if you are not sure!

So I think the only way to recover in your case is the chantools command:

https://github.com/lightninglabs/chantools/blob/master/doc/chantools_sweepremoteclosed.md

because your SCB does not contain the channel anymore, so thats why a recovery via SCB does not work.
